ReactAndroid/src/main/java/com/facebook/react/common/TestIdUtil.java

```diff
@@ -0,0 +1,65 @@
+package com.facebook.react.common;
+
+import android.view.View;
+
+import com.facebook.react.common.annotations.VisibleForTesting;
+
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * Utility methods for managing testIDs.
+ */
+public class TestIdUtil {
+    private static final ConcurrentHashMap<String, Integer> TEST_IDS = new ConcurrentHashMap<>();
+    private static final int STARTING_INTERNAL_ID = 10;
```

Comment:
The integers generated by R.class are typically higher values.  It could really be any value so long as it's low.  Should I add a comment to state that this is an arbitrary low value to avoid collisions with R.class?